baseapp/abci_utils.go

```diff
@@ -112,19 +119,14 @@ func ValidateVoteExtensions(
 			return fmt.Errorf("failed to verify validator %X vote extension signature", valConsAddr)
 		}
 
-		sumVP = sumVP.Add(bondedTokens)
```

Comment:
this is such a footgun. bondedTokens is actually useless in the `ValidatorStore` interface.

Reply:
yea wasn't sure if updating the staking keeper's `BondedTokensAndPubKeyByConsAddr` would break anything elsewhere. alternatively looks like we could just use `ValidatorByConsAddr(context.Context, sdk.ConsAddress) (ValidatorI, error)`

Reply:
`ValidatorI` has the pubkey we want